### PR TITLE
Removed faker dependency

### PR DESCRIPTION
**Pull request by rodelgc**
### Changes proposed in this Pull Request:

Closes 233-gh-woocommerce/woocommerce-quality.

This PR replaces the use of `faker` with either hard-coded values, or the native JS function `crypto.randomUUID()`. Our e2e test framework currently use `faker` in two places:

1. In one of our e2e tests, namely `packages/js/e2e-core-tests/src/specs/merchant/wp-admin-settings-shipping-classes.test.js` where `faker` was used to generate random name, slug, and description of two shipping classes that this test creates as test data. It is safe to replace these `faker` values with hard-coded ones since the shipping classes will be deleted at the end of the test anyway as part of the cleanup process, thus preventing duplicate shipping classes from other tests.
2. In our simple product factory in `e2e-utils`, namely `packages/js/e2e-utils/src/factories/simple-product.js` where `faker` was used to set a default `name` and `regular_price` of the simple product. The `name` can be created using a simple `crypto.randomUUID()` call to maintain uniqueness, while the `regular_price` can be safely hard-coded to a certain value more than $5.00. It's necessary to pick an amount greater than $5.00 so that we would still end up with a positive amount on tests where coupon discounts are applied (by default, our coupon discounts are hard-coded to around $5.00 as seen [here](https://github.com/woocommerce/woocommerce/blob/48fa8ee88e40951c09de56c2b0a8fb730b98b5a7/packages/js/e2e-core-tests/src/specs/utils/coupons.js#L6))

The simple product factory is not being used directly by our e2e core tests. Instead, the factory is used by these 3 convenience functions. These convenience functions are the ones that are being accessed by our e2e tests:
- `createSimpleProduct`
- `createSimpleProductWithCategory`
- `createGroupedProduct`

These 3 convenience functions are used by a lot of our tests (between 40-50 tests I think).

### How to test the changes in this Pull Request:

1. Just make sure that all our e2e tests pass.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Changed: Removed `faker` dependency on 

### FOR PR REVIEWER ONLY:

* [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Review by zhongruige — DISMISSED** [review]
Thanks for jumping on this one @rodelgc!

LGTM--once we resolve the merge conflict this is all set to merge in.

**Comment by rodelgc** [comment]
thanks @zhongruige I'll resolve the conflicts then merge it later today.

**Comment by rodelgc** [comment]
PR is now synced with latest changes in `trunk`. Merge conflict now resolved. Merging this PR now as it's already been [approved earlier](#pullrequestreview-919486334).

**Pull request merged**
